torchtext/data/field.py

```diff
@@ -130,13 +130,11 @@ class Field(RawField):
         torch.cuda.LongTensor: int
     }
 
-    def __init__(
-            self, sequential=True, use_vocab=True, init_token=None,
-            eos_token=None, fix_length=None, tensor_type=torch.LongTensor,
-            preprocessing=None, postprocessing=None, lower=False,
-            tokenize=(lambda s: s.split()), include_lengths=False,
-            batch_first=False, pad_token="<pad>", unk_token="<unk>",
-            pad_first=False):
+    def __init__(self, sequential=True, use_vocab=True, init_token=None, eos_token=None, fix_length=None,
+                 tensor_type=torch.LongTensor, preprocessing=None, postprocessing=None, lower=False,
+                 tokenize=(lambda s: s.split()), include_lengths=False, batch_first=False, pad_token="<pad>",
+                 unk_token="<unk>", pad_first=False):
+        super().__init__(preprocessing, postprocessing)
```

Comment:
This line either shouldn't be present or should use the Python 2 `super(self, Field)` syntax.